src/LuaBody.cpp

```diff
@@ -394,6 +438,29 @@ static int l_body_find_nearest_to(lua_State *l)
 	return 1;
 }
 
+static int l_body_get_phys_radius(lua_State *l)
+{
+		Body *b = LuaObject<Body>::CheckFromLua(1);
+		LuaPush(l, b->GetPhysRadius());
+		return 1;
+}
```

Comment:
Double indentation?

(Yeah, I know, I only look at the pattern of the text, not actual code :frowning: )